examples/illustrations/existence_intervals/README.md

```diff
@@ -115,3 +115,101 @@ Some of the examples in the OWL-TIME specification include demonstrations of `ti
 * [5.4, "iCalendar"](https://www.w3.org/TR/owl-time/#iCal) instantiates the endurant "Abraham Lincoln" (node `_:TE-2`).
 * [5.6, "A Use Case for Scheduling"](https://www.w3.org/TR/owl-time/#scheduling) instantiates perdurants some teleconference and some meeting (nodes `ex:telecon` and `ex:meeting`, respectively).
 * [5.7, "Alignment of PROV-O with OWL-Time"](https://www.w3.org/TR/owl-time/#time-prov) states that `prov:Activity` and `prov:InstantaneousEvent` can be subclasses of `time:TemporalEntity` and `time:Instant`, respectively.
+
+
+## gUFO
+
+(gUFO version: [1.0.0](https://github.com/nemo-ufes/gufo/releases/tag/v1.0.0).)
+
+gUFO's topmost class that describes an object that exists in a period of time is "Concrete Individual" (`gufo:ConcreteIndividual`), defined as the disjoint union of "Endurants," "Events," and "Situations."
+
+* `gufo:Endurant` corresponds with UCO Issue 535's "Endurant".
+* `gufo:Event` corresponds with UCO Issue 535's "Perdurant".
+* `gufo:Situation` is left out of scope of this discussion.
+
+Two properties, "has begin point" and "has end point," assign the first and last instants that any concrete individual exists.  The properties have range `time:Instant`, inheriting the vocabulary and entailing the structures of OWL-Time as described above.
```

Comment:
I had to read the part about "instants" twice to understand it as "time instants". I'd suggest to add "time" to it. This applies also further down the text a few "times" :-)

Reply:
This was an intentional abstraction on my part.  Sometimes, an instant of time is represented with `time:Instant`.  Sometimes, an instant is represented with a timestamp literal.  (See line 134, "This is an endurant...")  And I wanted to talk about reification a bit on line 155 ("This is the same endurant...").  Do you prefer being more explicit as you suggested?

Reply:
Ah, fair enough.
In that case I'd suggest to make this explicit by using those exact "Sometimes A Sometimes B". Either in-line or as a footnote with an (*) at the end of the line. The latter would not break or complicate the reading.

Reply:
How about this:

> Concrete individuals have first and last instants in which they exist.  These instants are represented as `time:Instant` objects or as datatype literals (typed `xsd:date` or `xsd:dateTimeStamp`---and, pertinent to UCO, not `xsd:dateTime`).  Two properties, "has begin point" and "has end point," assign the first and last instants that any concrete individual exists as objects.  The properties have range `time:Instant`, inheriting the vocabulary and entailing the structures of OWL-Time as described above.

Also see the new extra comment below on date- vs. date-time.

Reply:
Added in `a29fc4e`.